fuel-tx/src/transaction/types/input.rs

```diff
@@ -566,23 +566,29 @@ impl Input {
         compute_message_id(sender, recipient, nonce, amount, data)
     }
 
-    pub fn predicate_owner<P>(predicate: P) -> Address
+    pub fn predicate_owner<P>(predicate: P, params: &ConsensusParameters) -> Address
```

Comment:
Hmm, maybe we need to notify other teams about that explicitly

Reply:
yep - it's a big change